src/__tests__/http-test.js

```diff
@@ -1932,6 +1932,37 @@ describe('test harness', () => {
         );
       });
 
+      it('should have access to the GraphQL context property', async () => {
```

Comment:
Looking at above test I think it's better to modify it similar to my suggestion about the example in README, instead of adding new test specifically for context.